packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -309,6 +309,11 @@ type BaseInputCommonProps = FormInputLabelProps &
      * ```
      */
     trailingButton?: React.ReactElement<LinkProps>;
+    /**
+     * Whether to use Text or Heading component for Input text
+     * @default text
+     **/
+    valueComponentType?: 'text' | 'heading';
```

Comment:
Don't have much context on this, what exactly was the usecase for this?

Reply:
So for displaying value of the input, we sometimes need it to have the styles of our "Text" component and other times styles of our "Heading" component. Text & Heading differ in sizes as well as font families. OTPInput needs the input value to be shown as the same styles as Heading component whereas other input components need the same style as Text component